src/features/embalm/stepContent/index.tsx

```diff
@@ -26,7 +26,7 @@ export function StepContent() {
     [Step.SetRecipient]: { component: <SetRecipientPublicKey />, title: 'Set Recipient' },
     [Step.CreateEncryptionKeypair]: { component: <CreateEncryptionKeypair />, title: '' },
     [Step.SelectArchaeologists]: { component: <SelectArchaeologists />, title: '' },
-    [Step.TotalRequiredArchaeologists]: { component: <TotalRequiredArchaegologists />, title: '' },
+    [Step.RequiredArchaeologists]: { component: <TotalRequiredArchaegologists />, title: '' },
```

Comment:
@DarksightKellar -- good call on this update -- we may want to rename the component as well (`<TotalRequiredArchaegologists/>`)